app/Services/ValidationService/ValidationRules.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Services;
+
+class ValidationRules
+{
+    const RULES = [
+        'post' => [
+            'author_id' => [
+                'required' => true,
+                'integer'  => true,
+                'exists'   => 'users,id'
+            ],
+            'slug'      => [
+                'required' => true,
+                'string'   => true,
+                'unique'   => 'posts',
+                'regex'    => '/^[a-zA-Z0-9_-]*$/'
+            ],
+            'title'     => [
+                'required' => true,
+                'string'   => true,
+            ],
+            'excerpt'   => [
+                'max' => 255,
+            ],
+            'status_id' => [
+                'required' => true,
+                'integer'  => true,
+//                'exists'   => 'post_statuses.id',
+            ],
+        ],
+    ];
+
+    public static function get(string $entityName): array
+    {
+        $resultRules = self::concatenateRules(self::RULES[$entityName]);
+
+        return $resultRules;
+    }
+
+    public static function except(string $entityName, array $rulesToExclude = []): array
+    {
+        $rules = self::RULES[$entityName];
+        foreach ($rulesToExclude as $ruleToExclude => $ruleValue) {
+            if ( ! empty($rules[$ruleToExclude]) && array_key_exists($ruleValue,
```

Comment:
Unlike WP code standards there shouldn't be an extra spaces around `!` in Laravel apps. Please check code-sniffer rules.